js/main.js

```diff
@@ -163,6 +163,12 @@ function createWindow() {
   });
 
   app.commandLine.appendSwitch('disable-features', 'OutOfBlinkCors')
+  
+  if (process.platform === "linux"){
+    app.commandLine.appendSwitch("enable-experimental-web-platform-features", true);
+  } else {
+    app.commandLine.appendSwitch("enable-web-bluetooth", true);
```

Comment:
Does seting enable-web-bluetooth to true cause issues on linux?

Can we just add the enable-experimental-web-platform-features, if linux instead?